<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сериалы</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ico.jpg</normaloff>ico.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #262626; 
}

QLabel {
  color: #d9d9d9; 
  font-weight: 600;
font-size:20px;
}

QLabel#heading {
  color: #f0f0f0; 
  font-size: 18px;
  margin-bottom: 10px;
}

QLabel#subheading {
  color: #f0f0f0;
  font-size: 12px;
  font-weight: normal;
  margin-bottom: 10px;
}

QLineEdit {
color: #fff;
  border-radius: 8px;
  border: 1px solid #4d4d4d;
  padding: 10px 20px;
font-size: 20px
}

QLineEdit:focus {
  border: 1px solid #990000;
}

QLineEdit::placeholder {
  color: #808080;
}


QPushButton {
  background-color: #ff0000; 
  color: #ffffff; 
  font-weight: 600;
font-size:20px;
  border-radius: 8px;
  padding: 10px 20px;
  margin-top: 10px;
  border: 2px solid #990000; 
}

QPushButton:hover,
QPushButton:focus {
  background-color: #cc0000; 
  border: 1px solid #ff6666; 
}

QListWidget{
font-size: 25px;
font-weight: bold;
}

QListWidget::item { 
  border: 2px solid #ff0000; 
  border-radius: 10px; 
  padding: 2px; 
  margin: 1px; 
  background-color: #ff0000; 
  color: #ffffff; 

}

QComboBox {
font-size: 20px;
  border-radius: 8px;
  border: 1px solid #4d4d4d;
  padding: 10px 20px;
  color: #ffffff;
  background-color: #262626; 
}

QComboBox:focus{
	border-color: #990000;
}

QComboBox::drop-down {
    background-color: #262626; 
}

QComboBox::down-arrow:on {
    top: 1px; 
}

QComboBox QAbstractItemView {
    background-color: #262626; 
    border: 1px solid #4d4d4d; 
    color: #ffffff; 
    selection-background-color: #007acc;
}

QComboBox::item {
    padding: 5px;
}

QComboBox::item:selected {
    background-color: #007acc;
}

QComboBox::item:hower {
    background-color: #990000;
}

QSpinBox {
  border-radius: 8px;
  border: 1px solid #4d4d4d;
  padding: 10px 20px;
  color: #ffffff; 
  background-color: #262626; 
font-size: 20px;
}

QTableWidget {
  background-color: #262626; 
  color: #ffffff; 
  gridline-color: #4d4d4d;
}

QTableWidget::item {
  padding: 5px;
}

QTableWidget::item:selected {
  background-color: #ff0000; 
  color: #ffffff; 
}

QHeaderView::section {
  background-color: #ff0000; 
  color: #ffffff; 
  padding: 5px; 
}

QTableWidget::horizontalHeader {
  background-color: #ff0000;
  color: #ffffff; 
  border: 1px solid #4d4d4d; 
}

QTableWidget::verticalHeader {
  background-color: #ff0000;
  color: #ffffff; 
  border: 1px solid #4d4d4d; 
}

QTableWidget::verticalHeader::section {
  padding: 5px; 
}

QTableWidget QTableCornerButton::section {
  background-color: #ff0000;
  border: 1px solid #4d4d4d; 
}

QTableWidget::item:selected:focus {
  border: 1px solid #990000; 
}

QTableWidget::item:selected:hover {
  background-color: #cc0000; 
  color: #ffffff;
}

QSlider::groove:horizontal {
    border: 1px solid #4d4d4d;
    height: 10px; 
    background: #262626;
    margin: 0px;
}

QSlider::handle:horizontal {
    background: #ff0000; 
    border: 1px solid #990000;
    width: 20px; 
    margin: -10px 0; 
    border-radius: 10px; 
}

QSlider::add-page:horizontal {
    background: #fff; 
}

QSlider::sub-page:horizontal {
    background: #ff0000;
}

QVBoxLayout {
    border:12px solid #ffffff; 
    background-color: #fffff;
}

QCheckBox {
    color: #ffffff;
 	spacing: 5px;
	margin-left: 10px; padding-left: 10px;
}

QCheckBox::indicator {
    width: 15px;
    height: 15px;
}

QCheckBox::indicator:unchecked {
    background-color: #262626; 
    border: 2px solid #ffffff; 
}

QCheckBox::indicator:checked {
    image: url(galc.png); 
     border: 2px solid #ff0000;
}

QCheckBox:disabled {
    color: #808080;
}

QCheckBox::indicator:disabled {
    border: 2px solid #808080;
}

QScrollArea{
	color: #ff0000;
}

QCalendarWidget {
    color: #ffffff; 
    background-color: #262626;
    border: 2px solid #4d4d4d; 
    selection-background-color: #007acc; 
}

QCalendarWidget QToolButton {
    color: #ffffff; 
}

QCalendarWidget QToolButton:hover {
    background-color: #2b2b2b;
}

QCalendarWidget QMenu {
    background-color: #262626;
}

QCalendarWidget QMenu::item {
    color: #ffffff; 
    background-color: #262626; 
}

QCalendarWidget QMenu::item:selected {
    background-color: #007acc; 
}

QCalendarWidget QWidget {
    alternate-background-color: #2b2b2b; 
}

QCalendarWidget QAbstractItemView:enabled {
    color: #ffffff; 
}

QCalendarWidget QAbstractItemView:disabled {
    color: #808080;
}

QCalendarWidget QTextEdit {
    background-color: #262626;
}</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
      <property name="gridSize">
       <size>
        <width>0</width>
        <height>51</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="batchSize">
       <number>97</number>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Список сериалов</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Смотрю</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Просмотрено</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Запланировано</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Настройки</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Список</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Смотрю</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Просмотренно</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Добавить сериал</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Название</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Жанр</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Криминал</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Военный</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Фэнтези</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Комедия</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Драма</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Мультфильм</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Аниме</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Детектив</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>История</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Мелодрама</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Фантастика</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Количество серий</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Количество сезонов</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Год выпуска</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Страна выпуска</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Картинка</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="text">
             <string>Выбрать</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Оценка</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Изменить</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Загрузить в файл</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QTableWidget" name="tableWidget_2"/>
        </item>
        <item row="1" column="0">
         <layout class="QFormLayout" name="formLayout_6">
          <property name="topMargin">
           <number>23</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_19">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Удалить сериал</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Название</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
